src/summary/documentation/dimensions-card.resource.ts

```diff
@@ -0,0 +1,51 @@
+import { openmrsObservableFetch } from "@openmrs/esm-api";
+import { forkJoin } from "rxjs";
+import { map } from "rxjs/operators";
+import { Dimensions } from "./dimensions";
+
+const HEIGHT_CONCEPT = "5090AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const WEIGHT_CONCEPT = "5089AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+
+export function getDimensions(patientId: string) {
+  return forkJoin({
```

Comment:
I did not know that you could pass in an object to rxjs `forkJoin`. Pretty cool!